static/docs/commands-reference/add.md

```diff
@@ -92,9 +90,7 @@ This way you bring data provenance and make your project reproducible.
 
 - `-v`, `--verbose` - displays detailed tracing information.
 
-- `-f`, `--file` - specify name of the DVC-file it generates. It should be
-  either `Dvcfile` or have a `.dvc` file extension (e.g. `data.dvc`) in order
-  for `dvc` to be able to find it later.
+- `-f`, `--file` - specify name of the DVC-file it generates.
```

Comment:
I think it does not hurt to keep some explanation in this case. Especially considering that it can be Dvcfile. Also, put a link and update the DVC-file format doc to explain the naming at the very beginning.

Reply:
Updated to include more of an explanation for `-f` (here as well as in `dvc run` and `dvc import` cmd refs) – see latest commit.

The link to the **DVC-File Format** already exists earlier in all 3 docs, and that guide does already explain the naming strategy in the initial paragraph :)